gettsim/pensions.py

```diff
@@ -104,63 +106,63 @@ def _rentenwert_from_2018(tb_pens, yr):
     # Hence, some calculations have been made
     # in the data preparation.
     # First the Lohnkomponente which depands on the wage development of last years.
```

Comment:
Delete these comments (do not state the obvious)

Reply:
Deleted!